feature-detects/mediastream.js

```diff
@@ -0,0 +1,12 @@
+define(['Modernizr'], function (Modernizr) {
+    Modernizr.addTest('mediarecorder', typeof MediaRecorder);
+    typeof MediaRecorder !== undefined;
+
+
+    if (MediaRecorder) {
+        return true;
+    }
+    else {
+        return false;
+    }
```

Comment:
This code is not necessary as the `typeof MediaRecorder` in line 2 already provides the `true` or `false` that the `addTest` function requieres, so just press backspace!